### ISSUE TITLE
Workspace for higher level API

### ISSUE THREAD

**Issue by pinkforest** [tokio-rs/io-uring#266]
Would you ne amenable for separating lower level API and higher level APIs ? e.g. in a workspace

I created [i-uring-epoll](https://crates.io/crates/io-uring-epoll) high level API crate f.ex. and I could upstream it.

**Comment by quininer** [comment]
We already have an high level API crate https://github.com/tokio-rs/tokio-uring

**Comment by pinkforest** [comment]
Yeah but that makes several opinions and does things in tokio way. io-uring doesn't necessarily require tokio.
